docker/config/test.env

```diff
@@ -20,3 +20,7 @@ database_password=aPassword
 
 # Django tests are meant to run with debug mode disabled.
 DEBUG=False
+
+# Use test buckets
+resource.boto.bucket_name=crashstats-test
+TELEMETRY_BUCKET_NAME=telemetry-test
```

Comment:
OMG. Test settings in the webapp are complicated. There are some settings here and some in `webapp-django/crashstats/settings/test.py`. I don't think these two have any effect on things, but I can't be sure. I'd much rather have one set of settings, but the ones in the `test.py` file override a bunch of other stuff that we can't set via the environment.

Anyhow, so I added these here, but they might not affect anything and at some point, we should revisit webapp test configuration.